spec/factories/mentorships.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :mentorship do
+    mentee_id { FactoryGirl.create(:user, :mentee, :not_on_waitlist, :any_stage_of_career, primary_industry: "Technology").id }
```

Comment:
it's usually nice to create a 'mentee' factory and then provided these defaults for user ... should make it possible to not do this inline here.  Maybe not.  Recommend experiment